src/daemon/daemon.cpp

```diff
@@ -1480,12 +1480,12 @@ try // clang-format on
             info->set_load(mpu::run_in_ssh_session(session, "cat /proc/loadavg | cut -d ' ' -f1-3"));
             info->set_memory_usage(mpu::run_in_ssh_session(session, "free -b | grep 'Mem:' | awk '{printf $3}'"));
             info->set_memory_total(mpu::run_in_ssh_session(session, "free -b | grep 'Mem:' | awk '{printf $2}'"));
-            info->set_disk_usage(mpu::run_in_ssh_session(
-                session,
-                "df --output=used $(sudo fdisk -l | grep 'Linux filesystem' | awk '{print $1}') -B1 | sed 1d"));
-            info->set_disk_total(mpu::run_in_ssh_session(
-                session,
-                "df --output=size $(sudo fdisk -l | grep 'Linux filesystem' | awk '{print $1}') -B1 | sed 1d"));
+            info->set_disk_usage(
+                mpu::run_in_ssh_session(session, "df --output=used $(sudo fdisk -l | grep 'Linux filesystem' | awk "
```

Comment:
I never really got all this. Can't we just say `df --output=used /home`? In core instances, that's the partition with free space. In others, it's just under the root partition.

Reply:
Using `/home` seems to work for ubuntu instances, older and newer core instances, and also appliances. Was able to simplify the original command a bit as well.